.github/workflows/ci.yml

```diff
@@ -24,36 +24,42 @@ jobs:
   build-images:
     strategy:
       matrix:
-        docker-platform: ["linux/arm64", "linux/amd64"]
-        images: ["rocky9.2-10-gpu", "rocky9.2-10-slim", "rocky9.2-10-cpu"]
+        architecture: ["arm64", "amd64"]
+        image: ["wolfi-base", "rocky9.2-10-gpu", "rocky9.2-10-slim", "rocky9.2-10-cpu"]
```

Comment:
Think we should replicate this setup for the OSS repo since we have rocky and wolfi images now

Reply:
not sure what do you mean exactly 🤔 introducing matrix to parametrize this job:
https://github.com/Unstructured-IO/unstructured/blob/f1cab248cea6ceeff415fe5e1112781811bb000e/.github/workflows/ci.yml#L503C23-L503C28

Reply:
Yeah that way users can pull whichever version they like. So if you don't care about libreoffice you can still get the wolfi version for ARM.